test/cookbooks/app_test/recipes/app3_postrun.rb

```diff
@@ -8,7 +8,10 @@
       times += 1
       sleep 5
       puts "Still waiting for code.mulgara.org to start... #{times * 5} seconds"
-      raise('Failed to start code.mulgara.org') if times > 30
+      if times > 30
```

Comment:
Let's lower this to just 1 minute. No need to wait around for 2.5 minutes.
```suggestion
      if times > 6
```